engines/python/setup/djl_python/chat_completions/chat_properties.py

```diff
@@ -12,7 +12,7 @@
 # the specific language governing permissions and limitations under the License.
 from typing import Optional, Union, List, Dict
 
-from pydantic.v1 import BaseModel, Field, validator, root_validator
+from pydantic import BaseModel, Field, validator, root_validator
```

Comment:
Are we ready to upgrade pydantic?

Reply:
@frankfliu Sorry I just saw your comment, we are installing pydantic 2.6 in the docker, I didn't upgrade it.